specification/sbi_cove.adoc

```diff
@@ -843,8 +851,59 @@ The possible error codes returned in `sbiret.error` are shown below.
 | SBI_ERR_FAILED        | The operation failed for unknown reasons.
 |===
 
+[#sbi_covh_promote_to_tvm]
+=== Function: COVE Host Promote to TVM (FID #7)
+[source, C]
+-----
+struct sbiret sbi_covh_promote_to_tvm(unsigned long fdt_addr,
+                                      unsigned long number_of_vcpus,
+                                      unsigned long tap_addr);
+-----
+This intrisic is used by the host to promote a VM to a TVM. It is primarily inteded for CoVE deployment 
+models that require the single-step TVM creation (e.g., see <<dep3>>). Deployment models that offer the multi-step TVM creation 
+(e.g., <<dep1>>) may, but are not required to, support this ABI as an additional mechanism for creating TVMs. 
+
+The `fdt_addr` is the 8-bytes aligned guest physical address of the flattened device tree (FDT). The `fdt_addr` is 
```

Comment:
typo for param 2 
also a q - does not the fdt describe the number of vcpus also? what if that and the param 2 do not agree?

Reply:
Passing the number of vcpus as a parameter could simplify the TSM implementation, but indeed might lead to consistency issues (mismatch with the content of FDT). Thus, I removed this parameter.